lexicons/app/bsky/unspecced/searchActorsSkeleton.json

```diff
@@ -0,0 +1,56 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.unspecced.searchActorsSkeleton",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Backend Actors (profile) search, returning only skeleton",
+      "parameters": {
+        "type": "params",
+        "required": ["q"],
+        "properties": {
+          "q": {
+            "type": "string",
+            "description": "search query string; syntax, phrase, boolean, and faceting is unspecified, but Lucene query syntax is recommended. For typeahead search, only simple term match is supported, not full syntax"
+          },
+          "typeahead": {
+            "type": "boolean",
+            "description": "if true, acts as fast/simple 'typeahead' query"
+          },
+          "limit": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 100,
+            "default": 25
+          },
+          "cursor": {
+            "type": "string",
+            "description": "optional pagination mechanism; may not necessarily allow scrolling through entire result set"
+          }
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["actors"],
+          "properties": {
+            "cursor": { "type": "string" },
+            "hitsTotal": {
+              "type": "integer",
+              "description": "count of search hits. optional, may be rounded/truncated, and may not be possible to paginate through all hits"
+            },
+            "actors": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "format": "did"
+              }
+            }
```

Comment:
Not terribly worried about it, but there could be some value in wrapping the did in an object, just so that there's an upgrade path in case we want to provide any additional/meta info with each result in the future.

Reply:
Dan had the same thoughts in earlier review, which is strong enough signal that i'll add that to these endpoints.